geomesa-kafka/geomesa-kafka-datastore/src/main/scala/org/locationtech/geomesa/kafka/KafkaDataStore.scala

```diff
@@ -69,6 +70,14 @@ class KafkaDataStore(override val zookeepers: String,
     super[ContentDataStore].dispose()
     super[KafkaDataStoreSchemaManager].dispose()
   }
+
+  def getKafkaSchema(typeName: String): SimpleFeatureType = getSchema(new NameImpl(typeName))
```

Comment:
Ah I see. Well would it be alright to use `getFeatureSource.getSchema` if I'm going that route?